### ISSUE TITLE
Pretrained model with lesser receptive field

### ISSUE THREAD

**Issue by neha2110** [facebookresearch/VideoPose3D#148]
Hi, any idea where can I get the pre-trained models (Human3.6M) with receptive fields of 27 and 81? 
Thanks!

**Comment by dariopavllo** [comment]
I don't think we have those lying around, but it should be pretty fast to train them from scratch.

**Comment by neha2110** [comment]
True. It's just that the dataset link is not working so I was wondering if I could get them. Do you think that the dataset will be up soon?
